src/Framework/App.Ref/src/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -147,9 +147,13 @@ This package is an internal implementation of the .NET Core SDK and is not meant
 
       <AspNetCoreReferenceDocXml Include="@(_ResolvedProjectReferencePaths->WithMetadataValue('IsReferenceAssembly', 'false')->'%(RootDir)%(Directory)%(FileName).xml')" />
       <AspNetCoreReferenceDocXml Include="@(_SelectedExtensionsRefAssemblies->'$(MicrosoftInternalExtensionsRefsPath)%(FileName).xml')" />
+      <!-- Grab remaining .xml files from packages -->
+      <AspNetCoreReferenceDocXml 
+          Include="@(AspNetCoreReferenceAssemblyPath->WithMetadataValue('ExternallyResolved', 'true')->'%(RootDir)%(Directory)%(Filename).xml')" 
+          Condition="Exists('%(RootDir)%(Directory)%(Filename).xml')" />
 
       <RefPackContent Include="@(AspNetCoreReferenceAssemblyPath)" PackagePath="$(RefAssemblyPackagePath)" />
-      <RefPackContent Include="@(AspNetCoreReferenceDocXml)" PackagePath="$(RefAssemblyPackagePath)" />
+      <RefPackContent Include="@(AspNetCoreReferenceDocXml->Distinct())" PackagePath="$(RefAssemblyPackagePath)" />
```

Comment:
I'm less concerned about duplicate XML doc file paths than duplicate files in the final layout i.e. different doc files depending on which is copied second. Suggest copying `%(Filename)` from this item group into a temporary one and adding an `<Error/>` if its `->Count()` doesn't match `->Distinct()->Count()`.

Reply:
There will be an error anyways if there are duplicates, which will have a more descriptive error message than our custom error, since it lists the actual duplicate (similar to the .pdb issue).